bin/pinto

```diff
@@ -17,6 +17,8 @@ use warnings;
 my $home_var = 'PINTO_HOME';
 my $home_dir = $ENV{$home_var};
 
+$ENV{GNUPGHOME} = $ENV{PINTO_GNUPGHOME} if $ENV{PINTO_GNUPGHOME};
```

Comment:
> Or should I push it down to the 2 routines that call Module::Signature

Probably that. I always aim to keep side-effects in the smallest scope possible.